src/migrations/00097-backfill-device-service-on-device-service-environment-variable.async.ts

```diff
@@ -0,0 +1,36 @@
+import type { Migrator } from '@balena/pinejs';
+
+const migration: Migrator.AsyncMigration = {
+	asyncSql: `\
+		UPDATE "device service environment variable" AS dsev
+		SET
+			"device" = si."device",
+			"service" = si."installs-service"
+		FROM "service install" AS si
+		WHERE dsev."service install" = si."id"
+		AND dsev.id IN (
```

Comment:
NIT but other than that lgtm :+1: 

```suggestion
		AND dsev."id" IN (
```